scrapers/mt/bills.py

```diff
@@ -206,7 +206,7 @@ def _get_bill_status(self, status_page):
         status_text = status_page.xpath("//strong[text()='Current Bill Progress: ']/../text()")[0]
         return status_text.strip()
 
-    def parse_bill_status_page(self, url, page, session):
+    def parse_bill_status_page(self, url, page, list_sponsor, session):
```

Comment:
This was added and used in https://github.com/eidietrich/mtleg-bill-scraper/commit/f35406e372cbb6232ac459a7f169272179d1a355#diff-c242d880bb7570fc23288cded1279b8a4cc366619642b4d993528c76c2ef1facR136 , but this function signature wasn't updated.